aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -1304,7 +1304,7 @@ TORCH_IMPL_FUNC(mean_out)
   // (mean_kernel_impl()) is unvectorized and leads to very poor performance
   // for production workloads. Once that's fixed, the following code can be used
   // in lieu of the sum + divide implementation below.
-  if (self.device().is_cpu()) {
+  if (self.device().is_cpu() && dtype != kHalf && dtype != kBFloat16) {
```

Comment:
How about adding a note here explaining why you exclude bfloat16 and half here?

Reply:
Yes, added.